tests/integration/test_bayesian_optimization.py

```diff
@@ -217,11 +221,75 @@ def test_optimizer_with_vgp_model(use_natgrads: bool) -> None:
 
 
 @random_seed
-def test_optimizer_with_svgp_model() -> None:
+@pytest.mark.slow
+def test_bayesian_optimizer_with_svgp_finds_minima_of_scaled_branin() -> None:
+    acquisition_rule: AcquisitionRule[TensorType, SearchSpace] = EfficientGlobalOptimization()
+    _test_optimizer_finds_minimum(70, acquisition_rule, optimize_branin=True, model_type="SVGP")
+
+
+@random_seed
+def test_bayesian_optimizer_with_svgp_finds_minima_of_simple_quadratic() -> None:
     acquisition_rule: AcquisitionRule[TensorType, SearchSpace] = EfficientGlobalOptimization()
     _test_optimizer_finds_minimum(5, acquisition_rule, model_type="SVGP")
 
 
+@random_seed
+@pytest.mark.slow
+@pytest.mark.parametrize("num_steps, acquisition_rule", [(15, DiscreteThompsonSampling(1000, 3))])
+def test_bayesian_optimizer_with_dgp_finds_minima_of_scaled_branin(
+    num_steps: int, acquisition_rule: AcquisitionRule[TensorType, SearchSpace], keras_float: None
+) -> None:
+    _test_optimizer_finds_minimum(
+        num_steps, acquisition_rule, optimize_branin=True, model_type="DGP"
+    )
+
+
+@random_seed
+@pytest.mark.parametrize("num_steps, acquisition_rule", [(5, DiscreteThompsonSampling(1000, 1))])
+def test_bayesian_optimizer_with_dgp_finds_minima_of_simple_quadratic(
+    num_steps: int, acquisition_rule: AcquisitionRule[TensorType, SearchSpace], keras_float: None
+) -> None:
+    _test_optimizer_finds_minimum(num_steps, acquisition_rule, model_type="DGP")
+
+
+@random_seed
+@pytest.mark.slow
+@pytest.mark.parametrize(
+    "num_steps, acquisition_rule",
+    [
+        (80, EfficientGlobalOptimization()),
+        (30, DiscreteThompsonSampling(500, 3)),
+        # (10, DiscreteThompsonSampling(500, 3, thompson_sampler=ThompsonSamplerFromTrajectory())),
```

Comment:
?

Reply:
this waits for the keras trajectory sampling PR